### ISSUE TITLE
Help on Sync issues

### ISSUE THREAD

**Issue by cetiberiojr** [tidusjar/Ombi#3329]
**Describe the bug**

Hi, I have 2 servers running for the same content (Plex and Emby)... I figure out how to update all the informations on Ombi and so on.
BUT, I had a problem with "Watchmen" specifically, the metadata on one of servers were wrong (wrong series id), so I have this in my ombi: (Screenshot)

Some episodes in one, some in another (the correct is the first, the Watchmen from 2019).
When I run the sync from Emby, the correct is not getting all the episodes (its updated there already) and the wrong one is not losing the "Available" status...

How can I fix this little issue?
Also, we don't have discord or nothing like that to ask those quick questions?

**To Reproduce**
Running the Sync for Embi is not getting the series anymore.

**Expected behavior**
The serie synced correctly.

**Screenshots**
![image](https://user-images.githubusercontent.com/59163902/71645813-b6c41900-2cbc-11ea-919c-88ef15656300.png)

**Logs (Logs directory where Ombi is located)**
```
2020-01-01 20:18:19.221 +00:00 [Debug] Calling Execute on job System.IRefreshMetadata
2020-01-01 20:18:19.222 +00:00 [Information] Starting the Metadata refresh
2020-01-01 20:18:19.223 +00:00 [Information] The media item "The First Temptation of Christ" does not have a ImdbId, searching for ImdbId
2020-01-01 20:18:19.224 +00:00 [Information] The show "The First Temptation of Christ" has TheMovieDbId but not ImdbId, searching for ImdbId
2020-01-01 20:18:19.227 +00:00 [Debug] Trigger instruction : DeleteTrigger
2020-01-01 20:18:19.228 +00:00 [Debug] Deleting trigger
2020-01-01 20:18:19.246 +00:00 [Error] StatusCode: NotFound, Reason: Not Found, RequestUri: http://api.themoviedb.org/3/tv/652020/external_ids?api_key=b8eabaf5608b88d0298aa189dd90bf00
2020-01-01 20:18:19.261 +00:00 [Debug] Trigger instruction : DeleteTrigger
2020-01-01 20:18:19.261 +00:00 [Debug] Deleting trigger
```

This is to show the Watchmen is not getting picked up anymore, even after the update of metadata on Emby.

**Desktop (please complete the following information):**
Ombi on Unix Server machine

**Ombi Version (please complete the following information):**
- Version: 3.0.4892
- Media Server: Plex and Emby

**Comment by tidusjar** [comment]
Can you please expand? I don't fully understand the issue here.

Also why are you running both media servers?

**Comment by cetiberiojr** [comment]
Yes, I think that I poorly explained, hehe.

First; why I have both media servers:
I have both because some friends don't want to pay the Plex app on smartphone, so, they use Emby there, that is free.
Is also good to note, that I don't look at Emby very frequently, both media server are pointing to same folder, so I just hope that everything is going ok on Emby (Plex I use in my day to day, so I can check if something is wrong easily).

We have 2 Watchmen series on Ombi (Watchmen and Watchmen: The Motion Comic):

![image](https://user-images.githubusercontent.com/59163902/71730016-e0498580-2e1f-11ea-9c87-58f9db53752a.png)

I had in my servers, before installing Ombi the first one, series from 2019.
When I run the first sync, I just used the PLEX server sync and everything went ok BUT the series, none of the series got picked up, SO I configured the Emby also, and ran the Sync there, everything from Series now got synced and ok, when I looked at Watchmen, I saw this behaviour:

![image](https://user-images.githubusercontent.com/59163902/71730425-dbd19c80-2e20-11ea-9b24-9a73ca859026.png)

![image](https://user-images.githubusercontent.com/59163902/71730443-ed1aa900-2e20-11ea-9838-1a248ff3547a.png)

When I went to Emby, I see that indeed, the metadata there got the Motion Comic from 2008 (this was wrong, so I updated the metadata there to match the correct series (2019 one) and NOW we are at the problem.

Even if I run the manual Sync, I'm not getting the correct representation (all 2019 series, none of 2008 series).
Those prints are from now, and still we have the wrong representation on Ombi. 

How can I fix this issue?
Shouldn't Ombi check what I have and update accordling?
If I delete one series from my media servers they will be always be represented as available at Ombi?

Hope that it got a little more clear

Thanks!
